state/execution.go

```diff
@@ -45,6 +46,30 @@ type BlockExecutor struct {
 	metrics *Metrics
 }
 
+type CommitStepTimes struct {
+	CommitExecuting  types.StepDuration
+	CommitCommitting types.StepDuration
+	CommitRechecking types.StepDuration
+	Current          *types.StepDuration
+}
+
+func (st *CommitStepTimes) ToNextStep(from, next *types.StepDuration) time.Time {
+	now := time2.Now()
```

Comment:
About the alias name of the package. An alias name that makes sense, such as `canonictime`, is better than `time2`.

Reply:
Okay.